packages/accounts-password/password_server.js

```diff
@@ -516,17 +516,17 @@ Meteor.methods({forgotPassword: function (options) {
   Accounts.sendResetPasswordEmail(user._id, caseSensitiveEmail);
 }});
 
-// send the user an email with a link that when opened allows the user
-// to set a new password, without the old password.
-
 /**
- * @summary Send an email with a link the user can use to reset their password.
+ * @summary Creates options for email sending for reset password and enroll account emails.
+ * You can use this function when customizing a reset password or enroll account email sending.
  * @locus Server
  * @param {String} userId The id of the user to send email to.
- * @param {String} [email] Optional. Which address of the user's to send the email to. This address must be in the user's `emails` list. Defaults to the first email in the list.
+ * @param {String} email Which address of the user's to send the email to. This address must be in the user's `emails` list. If `null`, defaults to the first email in the list.
+ * @param {String} reason String `resetPassword` or `enrollAccount`.
```

Comment:
Yea, I do not like this duality between constants, but this is what we have now.